lib/libmbedtls/core/tee_lmd_provider.c

```diff
@@ -2071,6 +2151,58 @@ TEE_Result crypto_cipher_init(void *ctx, uint32_t algo,
 	const mbedtls_cipher_info_t *cipher_info = NULL;
 	int lmd_res;
 	int lmd_mode;
+#if defined(CFG_CRYPTO_CTS)
+	struct tee_symmetric_cts *cts;
```

Comment:
These helper variables are only used in their if blocks below so please move the declarations.

Reply:
OK.